metomi/isodatetime/main.py

```diff
@@ -49,12 +49,7 @@
 
     # 3.  Print ISO8601 duration as total amount of a unit
```

Comment:
Suggestion
> #3. Print ISO8601 duration as a number of of hours (H), minutes (M) or second (S)

do we need the 3.1 line given that there is only one section here?

Is there a reason that you have got rid fo the examples?